plugins/com.google.cloud.tools.eclipse.appengine.localserver.test/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,8 @@ Bundle-Vendor: Google Inc.
 Bundle-Version: 0.1.0.qualifier
 Fragment-Host: com.google.cloud.tools.eclipse.appengine.localserver
 Require-Bundle: com.google.cloud.tools.eclipse.test.dependencies
-Import-Package: com.google.cloud.tools.eclipse.swtbot,
+Import-Package: com.google.api.services.iam.v1;version="1.23.0",
```

Comment:
This is already imported in the main bundle. See if you can remove this. Otherwise, the version string should be "[1.23.0,1.24.0)".

Reply:
done